modules/fts/pages/fts-searching-with-sdk.adoc

```diff
@@ -13,54 +13,57 @@ Couchbase provides several SDKs to allow applications to access a Couchbase clus
 |===
 | SDK | Details | Link
 
-| Java SDK
-| The Java SDK forms the cornerstone of our JVM clients.
-It allows Java applications to access a Couchbase Server cluster.
-The Java SDK offers traditional synchronous APIs and scalable asynchronous APIs to maximize performance
-|xref:3.1@java-sdk:hello-world:overview.adoc[Java SDK 3.2]
-
-| Scala SDK
-| Our new Scala SDK allows Scala applications to access a Couchbase Server cluster.
-It offers synchronous, asynchronous, and reactive APIs for flexibility and maximum performance.
-|xref:1.1@scala-sdk:hello-world:overview.adoc[Scala SDK 1.2]
-
 | .NET SDK
 | The .NET SDK enables you to interact with a Couchbase Server cluster from the .NET Framework using any Common Language Runtime (CLR) language, including C#, F#, and VB.NET. 
 It offers both a traditional synchronous API and an asynchronous API based on the Task-based Asynchronous Pattern (TAP).
 []
```

Comment:
Not sure what the empty square brackets are for. Looks like you could just remove this line.

Reply:
:+1: